sdk/storage/azure-storage-common/src/test/java/com/azure/storage/common/policy/RequestRetryPolicyTest.java

```diff
@@ -0,0 +1,300 @@
+package com.azure.storage.common.policy;
```

Comment:
Build is failing for CI - Analyze because the copyright is missing. 
Add the following:

`// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.`